### PR TITLE
Implemented required classes.

### PR DESCRIPTION
**Pull request by muneebulrehman**
# Added 3 classes as below.
- `Person` class
- `Student` class which extends `Person` class
- `Teacher` class which extends `Person` class

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require relative './person.rb'
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom, name, age, can_use_services)
+    super(name, age, can_use_services)
```
Comment (dasileker) [inline student.rb]: - [x] Nice job however the `can_use_services` parameter is not correct instead you should use the **parent_permission**

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @muneebulrehman,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**
- Codes are clean ✅.
- Linters are passing ✅.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- The Readme file is having in the Getting started section `load 'main.rb` file but you still do not yet have that file, so please do not confuse the reader please remove it. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 1 above)
Comment (muneebulrehman) [reply student.rb]: My bad. I misread it. Thanks for pointing it out.

**Review by Emmyn5600 — APPROVED** [review]
Hi @muneebulrehman 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada: :clap: :clap: :100:

nodding-yes

## Optional suggestions

- [ ] N/A.

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by muneebulrehman** [comment]
Thanks @Emmyn5600

**Pull request merged**